samples/multi-kotlin-project-with-buildSrc/core/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    kotlin("jvm")
+    kotlin("jvm") version "1.1.51"
```

Comment:
:x: This introduces the version string twice in this sole sample, better request without applying in the root project and request with no versions in sub-projects